edenai_apis/apis/openai/openai_video_api.py

```
@@ -0,0 +1,107 @@
+            )
+
+        provider_job_id = provider_job_response["id"]
+
+        return AsyncLaunchJobResponseType(provider_job_id=provider_job_id)
+
+    def video__generation_async__get_job_result(
+        self,
+        provider_job_id: str,
+    ) -> AsyncBaseResponseType[GenerationAsyncDataClass]:
+        get_video_url = f"https://api.openai.com/v1/videos/{provider_job_id}"
+        response = requests.get(url=get_video_url, headers=self.headers)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)